benchmarks/dashboard-stack/lambda/benchmarkManager.py

```diff
@@ -52,9 +54,29 @@ def benchmarkManager(event, context):
                 check your consol to see it succeed or not'.format(stack_name)
         }
     elif event['action'] == 'test':
-        # trigger codebuild to deploy the benchmarks stack
         codebuild = boto3.client('codebuild')
-        response = codebuild.start_build(projectName=CODE_BUILD_NAME)
+        if "project_name" not in event:
```

Comment:
I think we should eventually add an "action" that's part of the event that makes it explicit what the caller is asking the Lambda to do.  Could just add a TODO for now.